checker/src/summaries.rs

```diff
@@ -94,6 +94,30 @@ pub struct Summary {
     pub unwind_side_effects: Vec<(Path, AbstractValue)>,
 }
 
+impl PartialEq for Summary {
```

Comment:
Not a fan of a single big return a == b && this == that and so on, eh?

Reply:
It really helps with debugging when you can set break points at particular parts of the expression.